.github/workflows/ci.yml

```diff
@@ -166,8 +166,8 @@ jobs:
         id: cypress-run
         uses: cypress-io/github-action@v5.8.3
         with:
-          wait-on: http://localhost:3000
-          wait-on-timeout: 200
+          wait-on: http://[::1]:3000
+          wait-on-timeout: 600
```

Comment:
CI tests passed. What was the issue? how did you end up with this "http://[::1]"?
https://github.com/cowprotocol/cowswap/actions/runs/7460594214/job/20298965268?pr=3590

Reply:
https://github.com/cypress-io/github-action/issues/634#issuecomment-1405967020

Reply:
https://github.com/cypress-io/github-action/blob/master/README.md#wait-on-with-nodejs-18